thrift/src/main/thrift/atoms/explainer.thrift

```diff
@@ -2,16 +2,59 @@ namespace * contentatom.explainer
 namespace java com.gu.contentatom.thrift.atom.explainer
```

Comment:
As a not in this file - anything you mark as required in this file will have to be filled in by someone in the tool as the UI for the tool is generated from this model. Are you sure that everything you mark as required is actually required?